webapp/app/[locale]/stake/_components/tokenRewards.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react'
+
+import { HemiTag } from './rewardTag'
+
+interface Props {
+  rewards: string[]
+}
+
+// TODO: It's pending to implement some missing tags
+// Issue #799 https://github.com/hemilabs/ui-monorepo/issues/799
+const rewardComponentMap: Record<string, ReactNode> = {
+  hemi: <HemiTag />,
+}
```

Comment:
Same as [this comment](https://github.com/hemilabs/ui-monorepo/pull/800/files#r1935646330)

```suggestion
const rewardComponentMap: Record<StakeToken['extensions']['rewards'], ReactNode> = {
  hemi: <HemiTag />,
}
```

Plus, here it would prevent you from adding incorrect reward names if `rewards` is not typed as a string (See [this comment first](https://github.com/hemilabs/ui-monorepo/pull/800/files#r1935652293))

Reply:
Done. thanks